src/Microsoft.Diagnostics.NETCore.Client/DiagnosticsIpc/IpcClient.cs

```diff
@@ -19,45 +20,72 @@ internal class IpcClient
         /// </summary>
         /// <param name="endpoint">An endpoint that provides a diagnostics connection to a runtime instance.</param>
         /// <param name="message">The DiagnosticsIpc Message to be sent</param>
-        /// <returns>The response DiagnosticsIpc Message from the dotnet process</returns>
-        public static IpcMessage SendMessage(IpcEndpoint endpoint, IpcMessage message)
+        /// <returns>An <see cref="IpcResponse"/> containing the response message and optional continuation stream.</returns>
+        public static IpcResponse SendMessage(IpcEndpoint endpoint, IpcMessage message)
```

Comment:
I think this is a good way to share validation code and such, but i'm wondering if there should still be an overload that just turns a message, so that the caller doesn't have to worry about disposing the stream